yarn.lock

```diff
@@ -29178,6 +29178,18 @@ tar@^6.0.2, tar@^6.1.11, tar@^6.1.15, tar@^6.1.2:
     mkdirp "^1.0.3"
     yallist "^4.0.0"
 
+tar@^6.2.1:
```

Comment:
Question: looking at the `^` semvers, can all these `tar` be consolidated with `^6.2.1`?

Reply:
Yup! Done. 👍